lib/googleauth/service_account.rb

```diff
@@ -94,6 +101,7 @@ def self.unescape str
       def initialize options = {}
         @project_id = options[:project_id]
         @quota_project_id = options[:quota_project_id]
+        @enable_self_signed_jwt = options[:enable_self_signed_jwt] ? true : false
```

Comment:
We want self-signed JWTs to be enabled by default... this implies it defaults to disabled, yes?

Reply:
Yes, at this layer it defaults to disabled. See the main PR description. The intent is for the generated clients to set this to enabled if they determine it is safe (i.e. that the endpoint is the service's default). The end user should not need to touch this.